kidkode/src/main/webapp/test_quiz_style.css

```diff
@@ -106,13 +115,35 @@ label {
   padding: 16px 32px;
   text-decoration: none;
 }
-#submit:hover {
+
+.button:hover {
   background-color: rgb(0, 173, 144);
   border-color: rgb(0, 127, 106);
 }
 
-#submit:active {
+.button:active {
   border-width: 3px;
   position: relative;
   top: 5px;
 }
+
```

Comment:
I've added styling for the `slide` (where each question and choice is being nested in) and `quiz-container` (where all questions are being nested in) here. I suspect this needs to be adjusted to fix the bug?

Reply:
This works, thanks so much! I also managed to fix the button-clicking issue by making its position relative and z-index higher than the question container! 👍